src/main/java/org/fundacion/jala/converter/models/parameter/VideoParameter.java

```diff
@@ -4,15 +4,16 @@
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
- */
-/**
- * @author Daniela Santa Cruz
- * @colaborathor Paola Aguilar
+ *
+ * @author Daniela Santa Cruz Andrade
  */
 package org.fundacion.jala.converter.models.parameter;
 
 import org.springframework.stereotype.Service;
 
+/**
+ * This class is the options of the audio parameters.
```

Comment:
Review the JavaDoc, maybe is better, are the options for the audio....

Reply:
ok